.github/ISSUE_TEMPLATE/beta-minor-version-release.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Beta minor version release
+about: Creates a tracking checklist of items for a Beta minor version release
+title: "[Tracking] v#.##.#B# release "
+labels: 'release'
+assignees: ''
+
+---
+
+### Release Core
+- [ ] [Engineering] Follow [dbt-release workflow](https://www.notion.so/dbtlabs/Releasing-b97c5ea9a02949e79e81db3566bbc8ef#03ff37da697d4d8ba63d24fae1bfa817) 
+- [ ] [Engineering] Verify new release branch is created in the repo
+- [ ] [Product] Finalize migration guide (next.docs.getdbt.com)
+
+### Release Cloud
+- [ ] [Engineering] Create a platform issue to update dbt Cloud and verify it is completed. [Example issue](https://github.com/dbt-labs/dbt-cloud/issues/3481)
+- [ ] [Engineering] Determine if schemas have changed. If so, generate new schemas and push to schemas.getdbt.com
```

Comment:
I'm not sure if we should be generating new schemas at beta-time, in case they change again between the beta and the RC. That said, I do think it makes sense to give folks (especially our Metadata team) a heads up with the changes so far.

Options:
- Start using prerelease suffixes for our artifact schemas... but that might be overkill
- Bump the versions on artifacts that have already changed, and make it _very very clear_ that those version numbers are not final and those artifacts might still change again

To be clear: Our long-term objective should be to make all artifact-mutating changes before releasing the first beta of a new minor version

Reply:
I would agree that our ideal state would be that we don't make schema changes after we cut the beta (or else we cut a new beta). I'm going to leave this as is but will talk to the Metadata team about what they think of beta vs rc vs final releases. I'm curious on their input

Reply:
> or else we cut a new beta

also fair!